beacon/endpoints/rest/schemas/default.py

```diff
@@ -31,66 +37,83 @@ def beacon_info_v20(datasets, authorized_datasets=[]):
         'entryPoint': conf.entry_point,
         'open': conf.is_open,
-        'datasets': [beacon_dataset_info_v20(row, authorized_datasets) for row in datasets],
+        'datasets': [beacon_dataset_info_v30(row, authorized_datasets) for row in datasets],
         'info': None,
     }
 
 
-def beacon_dataset_info_v20(row, authorized_datasets=[]):
-    dataset_id = row['datasetId']
+def beacon_dataset_info_v30(row, authorized_datasets=[]):
+    dataset_id = row['stable_id']
     is_authorized = dataset_id in authorized_datasets
 
     return {
         'id': dataset_id,
         'name': row['name'],
         'description': row['description'],
-        'assemblyId': row['assemblyId'],
-        'createDateTime': row['createdAt'].strftime(conf.datetime_format) if row['createdAt'] else None,
-        'updateDateTime': row['updatedAt'].strftime(conf.datetime_format) if row['updatedAt'] else None,
+        'assemblyId': row['reference_genome'],
+        'createDateTime': row['created_at'].strftime(conf.datetime_format) if row['created_at'] else None,
+        'updateDateTime': row['updated_at'].strftime(conf.datetime_format) if row['updated_at'] else None,
         'dataUseConditions': None,
         'version': None,
-        'variantCount': row['variantCount'],  # already coalesced
-        'callCount': row['callCount'],
-        'sampleCount': row['sampleCount'],
+        'variantCount': row['variant_count'],
+        'callCount': row['call_count'],
+        'sampleCount': row['sample_count'],
         'externalURL': None,
+        'handovers': row['handovers'],
         'info': {
-            'accessType': row['accessType'],
-            'authorized': True if row['accessType'] == 'PUBLIC' else is_authorized,
-            'datasetSource': row['datasetSource'],
-            'datasetType': row['datasetType']
+            'accessType': row['access_type'],
+            'authorized': True if row['access_type'] == 'PUBLIC' else is_authorized,
+            'datasetSource': row['dataset_source'],
+            'datasetType': row['dataset_type']
         }
     }
 
 
-def beacon_variant_v20(row):
+def beacon_variant_v30(row):
     return {
             'variantId': row['variant_id'],
+            'assemblyId': row['assembly_id'],
             'refseqId': row['refseq_id'],
+            'start': row['start'],
+            'end': row['end'],
             'ref': row['reference'],
             'alt': row['alternate'],
             'variantType': row['variant_type'],
-            'start': row['start'],
-            'end': row['end'],
-            'assemblyId': row['assembly_id'],
-            'info': None,
+            'info': {
+                'chromosome': row['chromosome'],
+                'effect_impacts': row['effect_impacts'],
+                'functional_classes': row['functional_classes'],
+                'genomic_regions_ontology': row['genomic_regions_ontology'],
+                'molecular_effects_ontology': row['molecular_effects_ontology'],
```

Comment:
This is not the correct place to show this information about annotations (effect_impacts, functional_classes, genomic_regions_ontology, molecular_effects_ontology). Remove it, please!